src/styles/QuestionAnswersStyles.scss

```diff
@@ -0,0 +1,44 @@
+@import 'partials/variables';
+
+$correct-answer: rgb(10, 215, 10);
+
+#answers {
+  box-sizing: border-box;
+  margin: 1em auto;
+  grid-area: rules;
+}
+
+.answer {
+  font-size: 2rem;
+  font-family: 'Montserrat', sans-serif;
+  background: $white;
+  border: none;
+  border-radius: 10px;
+  outline: none;
+  width: 80%;
+  min-width: 10em;
+  padding: 0.5em 2em;
+  margin: 0.3em 0.5em;
+  transition: box-shadow 0.2s ease;
+  cursor: pointer;
+  &:not(.wrong-answer):not(.correct-answer):hover {
+    box-shadow: inset 0 0 7.5px #000;
+  }
+  @media (max-width: 640px) {
+    font-size: 1.5rem;
+  }
+}
+
+.correct-answer, .wrong-answer {
+  font-weight: 700;
```

Comment:
Wydaje mi się, że jak ten styl wrzucimy do selectora .answer to będzie działało tak samo, możesz sprawdzić

Reply:
.answer opisuje style dla każdej odpowiedzi(przed interakcją), font-weight: 700; nadaje dla odpowiedzi już udzielonych